cmd/qemu-ifdown/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/viper"
+)
+
+var DefaultConfPath string
+
+// todo: error handling
+
+func runCmd(cmd string, args []string) {
+	c := exec.Command(cmd, args...)
+	c.Run()
+}
+
+func init() {
+	viper.SetDefault("bridges.type", "linux")
+	viper.SetDefault("bridges.name", "br0")
+	viper.SetConfigFile(DefaultConfPath)
+	viper.SetConfigType("toml")
+	viper.AutomaticEnv()
+	err := viper.ReadInConfig()
```

Comment:
Basically ``func init()`` is not the place to make OS syscall operation. Please move to ``main()``.